makefile

```diff
@@ -6,18 +6,22 @@ SHELL := /bin/bash
 .PHONY: clean test integration consul etcd run-consul run-etcd example example-consul example-etcd ship dockerfile docker cover lint
 
 VERSION ?= dev-build-not-for-release
-LDFLAGS := '-X main.GitHash=$(shell git rev-parse --short HEAD) -X main.Version=${VERSION}'
+LDFLAGS := '-X containerbuddy.GitHash=$(shell git rev-parse --short HEAD) -X containerbuddy.Version=${VERSION}'
 
 ROOT := $(shell pwd)
+PACKAGE := github.com/joyent/containerbuddy
+GO15VENDOREXPERIMENT := 1
+export GO15VENDOREXPERIMENT
 
 COMPOSE_PREFIX_ETCD := exetcd
 COMPOSE_PREFIX_CONSUL := exconsul
 
+
 DOCKERMAKE := docker run --rm \
 	--link containerbuddy_consul:consul \
 	--link containerbuddy_etcd:etcd \
-	-v ${ROOT}/src/containerbuddy:/go/src/containerbuddy \
 	-v ${ROOT}/.godeps:/go/src \
```

Comment:
Did you mean `GO15VENDOREXPERIMENT` ? They dont reference GOVENDORPATH [anywhere in the code](https://github.com/tools/godep/search?utf8=%E2%9C%93&q=GO15VENDORPATH&type=Code)